APCAppCore/APCAppCore/UI/Onboarding/SignUp/APCAllSetContentViewController.m

```diff
@@ -109,24 +109,27 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     switch (indexPath.row) {
         case APCAllSetRowActivities:
         {
-            NSString *original = NSLocalizedString(@"You’ll find your list of daily surveys and tasks on the “Activities” tab. New surveys and tasks will appear over the next few weeks.",
-                                                   @"You’ll find your list of daily surveys and tasks on the “Activities” tab. New surveys and tasks will appear over the next few weeks.");
+            NSString *original = NSLocalizedString(@"You’ll find your list of daily surveys and tasks on the “Activities” tab. New surveys and tasks will appear over the next few weeks.", nil);
+
+            cell.originalText = original;
             
-            if (self.textBlocks) {
-                for (NSDictionary *textBlock in self.textBlocks) {
-                    if (textBlock[kAllSetActivitiesTextOriginal]) {
+            if (self.textBlocks)
+            {
+                for (NSDictionary *textBlock in self.textBlocks)
+                {
+                    if (textBlock[kAllSetActivitiesTextOriginal])
+                    {
                         cell.originalText = textBlock[kAllSetActivitiesTextOriginal];
-                    } else {
-                        cell.originalText = original;
                     }
                     
-                    if (textBlock[kAllSetActivitiesTextAdditional]) {
+                    if (textBlock[kAllSetActivitiesTextAdditional])
+                    {
                         cell.additonalText = textBlock[kAllSetActivitiesTextAdditional];
-                    } else {
-                        cell.additonalText = nil;
-                    }
+                    } 
                 }
-            } else {
+            }
+            else
+            {
                 cell.originalText = original;
```

Comment:
Seems this else block is unnecessary, see that `originalText` has been set on line 114.